docker/depends/pecan.depends

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# autogenerated do not edit
+# use scripts/generate.dockerfile.depends
+
+# stop on error
+set -e
+
+# Don't use X11 for rgl
+RGL_USE_NULL=TRUE
+
+# install remotes first in case packages are references in dependencies
+installGithub.r \
+    araiho/linkages_package \
+    ebimodeling/biocro \
+    MikkoPeltoniemi/Rpreles \
+    ropensci/geonames \
+    ropensci/nneo
+
+# install all packages (depends, imports, suggests)
+install2.r -e -s \
```

Comment:
Nice use of `littler`! This is a much better solution than what we had.

I would consider adding `-n -1`, which is a hidden option that tells this script to use as many cores as are available (https://github.com/eddelbuettel/littler/blob/master/inst/examples/install2.r#L66-L69).

Reply:
in travis we have a comment about some issues when using more than 1 cpu:

```
  # TODO: Would probably be faster to use -j2 NCPUS=1 as for other steps,
  # but many dependency compilations seem not parallel-safe.
  # More debugging needed.
```

Reply:
I added that comment in frustration several build revisions ago, and I don't remember which packages were causing trouble or how many "many" was. Probably worth revisiting at some point -- maybe the situation has improved?

Reply:
added -n -1